goodreads-predictor/src/goodreads_predictor/pipelines/feature_engineering/nodes.py

```diff
@@ -0,0 +1,233 @@
+"""
+This is a boilerplate pipeline 'feature_engineering'
+generated using Kedro 0.19.6
+"""
+from typing import Tuple, List, Union, Dict, Any
```

Comment:
Might change typing to typing_extensions ! As typing is built in for python 3.11 and plus.

Reply:
Done, thanks for the feedback!